hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocol/S3Auth.java

```diff
@@ -24,6 +24,7 @@
   private String stringToSign;
   private String signature;
   private String accessID;
+  public static final String S3_AUTH_CHECK = "ozone.s3.auth.check";
```

Comment:
This will be S3Gateway config, but it is internal should not be exposed to end users if i understand correctly.

Reply:
Yes that is correct. It is not exposed as a end user config.